docs/_manage/versions-cli.md

```diff
@@ -56,46 +56,45 @@ There are two strategies for this. You can “hide” the existing Trigger/Actio
+When ready, you can Promote your new integration version to make it the default version that Zapier shows to new users when creating new Zaps. For a complete guide to the `promote` command, look [here](../cli_docs/cli#promote).
 
-Major versions of integrations—especially if authentication or API calls were changed and other breaking changes were added—will require users to re-create their Zaps with the new version of your integration. In that case, you won’t be able to migrate existing users. Instead, you’ll need to promote the new version and deprecate the old one so new users and Zaps will be made with the new integration.
+If this isn’t the first time you’ve promoted your app - you might have users on older versions. You can `zapier migrate` to either move users over (which can be dangerous if you have breaking changes). Or, you can `zapier deprecate` to give users some time to move over themselves (which can be disruptive to user workflows).
 
 **Examples**
 
 ```
-# migrate 100% of your users between version 1.0.0 over to 1.0.1
-zapier migrate 1.0.0 1.0.1
+# promote your app version to all Zapier users
+zapier promote 1.0.1
 
-# migrate 15% of your users between version 1.0.1 over to 2.0.0
-zapier migrate 1.0.1 2.0.0 15
+# OPTIONAL - migrate your users between one app version to another
+zapier migrate 1.0.0 1.0.1
 
-# migrate the specific user user@example.com between version 2.0.0 to 2.0.1
-zapier migrate 2.0.0 2.0.1 --user=user@example.com
+# OR - mark the old version as deprecated
+zapier deprecate 1.0.0 2020-06-01
 ```
+## Migrate Users to Your New Integration Version
 
-## Promote a New Version of Your Integration
+Next, you can start rolling your updated integration out to users to update their existing Zaps and let users make new Zaps with your new integration features.
+
+If you made a patch version with only minor, non-breaking changes, you can migrate existing users and Zaps to your new integration version. For a complete guide to the `migrate` command, look [here](../cli_docs/cli#migrate).
 
-You can also Promote your new integration version to make it the default version that Zapier shows to users when creating new Zaps. For a complete guide to the `promote` command, look [here](../cli_docs/cli#promote).
+Typically, Zapier recommends moving a small percentage of your existing users—perhaps 10 or 20%—to the new integration version to make sure everything is working as expected. Once everything looks good, you can migrate the remainder of your users to the new version. In some cases, it may be more useful to migrate only a portion of the current users, to ensure that the minor change did not accidentally introduce any unforeseen issues. You can do that using the **Percentage** option in the Migration window, or select **Email** to migrate only one user at a time.
```

Comment:
This sentence seems redundant to the first sentence starting with "Typically"? Maybe we can advise them to monitor the logs in the "Monitoring" tab to watch for unforeseen issues?